osu.Game/Graphics/UserInterfaceV2/OsuHSVColourPicker.cs

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using JetBrains.Annotations;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Effects;
+using osu.Framework.Graphics.Shapes;
+using osu.Framework.Graphics.UserInterface;
+using osu.Game.Overlays;
+using osuTK;
+
+namespace osu.Game.Graphics.UserInterfaceV2
+{
+    public class OsuHSVColourPicker : HSVColourPicker
+    {
+        protected override HueSelector CreateHueSelector() => new OsuHueSelector();
+        protected override SaturationValueSelector CreateSaturationValueSelector() => new OsuSaturationValueSelector();
+
+        [BackgroundDependencyLoader(true)]
+        private void load([CanBeNull] OverlayColourProvider colourProvider, OsuColour osuColour)
+        {
+            Background.Colour = colourProvider?.Dark5 ?? osuColour.GreySeafoamDark;
+
+            Content.Padding = new MarginPadding(10);
+            Content.Spacing = new Vector2(0, 10);
+        }
+
+        private class OsuHueSelector : HueSelector
+        {
+            public OsuHueSelector()
+            {
+                Margin = new MarginPadding
+                {
+                    Bottom = 15
+                };
+
+                SliderBar.CornerRadius = SliderBar.Height / 2;
+                SliderBar.Masking = true;
+            }
+
+            protected override Drawable CreateSliderNub() => new SliderNub();
+
+            private class SliderNub : CompositeDrawable
+            {
+                public SliderNub()
+                {
+                    InternalChild = new Triangle
+                    {
+                        Width = 20,
+                        Height = 15,
+                        Anchor = Anchor.BottomCentre,
+                        Origin = Anchor.TopCentre
+                    };
+                }
+            }
+        }
+
+        private class OsuSaturationValueSelector : SaturationValueSelector
+        {
+            public OsuSaturationValueSelector()
+            {
+                SelectionArea.CornerRadius = 10;
+                SelectionArea.Masking = true;
+                // purposefully use hard non-AA'd masking to avoid edge artifacts.
+                SelectionArea.MaskingSmoothness = 0;
```

Comment:
The following is what happens if this is omitted (magnified 3x):

![2021-06-30-233138_108x100_scrot](https://user-images.githubusercontent.com/20418176/124034387-6e91ce00-d9fb-11eb-8e74-36115ea19a0a.png)

Unsure if there is a better way of getting rid.